.travis.yml

```diff
@@ -21,44 +21,100 @@ compiler:
   - gcc
```

Comment:
Looking at the current builds, I'm not sure Clang is ever used for integration tests. It looks like the Makefile would need modification for that to happen.